CMakeLists.txt

```diff
@@ -118,6 +123,18 @@ if(NOT DISABLE_TESTS)
   add_subdirectory(tests)
 endif()
 
+# Code formatting target
+find_program(CLANG_FORMAT "clang-format")
+if(NOT CLANG_FORMAT)
+  message(WARNING "clang-format not found! Target `format` will fail!")
+endif()
```

Comment:
It looks like a serious warning, but I think it should be optional for non-contributors. Can we make the message less exclaimful (is that a word?), for example "clang-format not found. Target `format` will not work." ?

Reply:
I will remove the message, it will print out clang_format_NOTFOUND when attempting to build `format`..should be enough